packages/samples/headless-ssr-commerce/app/(listing)/[category]/page.tsx

```diff
@@ -12,13 +12,32 @@ import Summary from '@/components/summary';
 import {listingEngineDefinition} from '@/lib/commerce-engine';
 import {NextJsNavigatorContext} from '@/lib/navigatorContextProvider';
 import {headers} from 'next/headers';
+import {notFound} from 'next/navigation';
+
+// This is a mapping of the Merchandising Hub urls to urls without slashes
```

Comment:
Ideally we'd get rid of this in the sample.

We can create listing page configurations that will have the correct URLs.

We don't have to do this in this PR, but I think it would be worth creating a Jira and adding a TODO.

Reply:
ok ill do it